src/tchannel_bridge.js

```diff
@@ -139,7 +139,7 @@ export default class TChannelBridge {
         return channel;
     }
 
-    _getTChannelParentSpan(): TChannelSpan {
+    static getTChannelParentSpan(): TChannelSpan {
```

Comment:
are you sure about this? This is a bad method name anyway, would be more accurate to call it "makeFakeTChannelParentSpan()", because we return 0 trace id.